types/koa/index.d.ts

```diff
@@ -453,32 +453,32 @@ declare class Application extends EventEmitter {
         port?: number,
         hostname?: string,
         backlog?: number,
-        listeningListener?: () => void,
+        listeningListener?: (err?: Error) => void,
```

Comment:
This listener eventually [gets registered](https://github.com/koajs/koa/blob/2180839eda2cb16edcfda46ccfe24711680af850/lib/application.js#L62-L66) with [Node's HTTP server's `"request"` event](https://github.com/nodejs/node/blob/16cffb0d4890bf4b4ff8ad0bbd7731105fabe845/lib/_http_server.js#L293), so is this really the right type? Shouldn't it get [a request and response](https://github.com/nodejs/node/blob/16cffb0d4890bf4b4ff8ad0bbd7731105fabe845/lib/_http_server.js#L650) or something? Kind of